experimental/generation/runbot/RunBot.csproj

```diff
@@ -28,14 +28,15 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="Iciclecreek.Bot.Builder.Dialogs.Recognizers.LuceneZY" Version="4.13.0" />
```

Comment:
>LuceneZY [](http://example.com/codeflow?start=75&length=8)

You can't publish the standard package?

Reply:
When the form-operations.en-us.data issue is resolved, I will publish the standard package. And e2e test it.

Reply:
I think for nuget.org, different onwers (tom and zoma) could not have the nuget pkg with the same name. So I rename the nuget pkg to Iciclecreek.Bot.Builder.Dialogs.Recognizers.Lucene.Dynamiclist.